_alerts/2017-09-28-segwit2x-safety.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Beware of Coinbase, Xapo, and Bitpay!"
+shorturl: "segwit2x-safety"
+active: true
+banner: "Beware of Coinbase, Xapo, and Bitpay!"
```

Comment:
Instead of naming specific companies in the title, I think the title should be more generic warning about companies and services that support segwit2x in general.